src/routes/agent.router.ts

```diff
@@ -67,5 +54,19 @@ agentRouter.post(
   AgentController.setOnCallBotFlow,
 );
 agentRouter.get("/on-call", AgentController.getOnCallStatus);
-
+agentRouter.get("/support", AgentController.getSupportNumbers);
+agentRouter.post(
+  "/support",
+  validateSupportRequest(),
+  checkExact(),
+  throwIfBadRequest,
+  AgentController.updateSupportNumbers,
+);
+agentRouter.post(
+  "/reset-player-password",
+  validateResetPasswordRequest(),
+  checkExact(),
+  throwIfBadRequest,
+  AgentController.resetPlayerPassword,
```

Comment:
**🚨 suggestion (security):** New password reset functionality needs security review

The addition of a password reset functionality for agents is a sensitive operation. Ensure that this endpoint is properly secured, audited, and only accessible to authorized agents. Consider implementing additional safeguards such as requiring secondary authentication for this operation.